### PR TITLE
Crux SaaS node express server init

### PR DESCRIPTION
**Pull request by SanchayPahalwani**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
+    "build-and-start-saas-server": "./node_modules/.bin/parcel build ./src/application/clients/crux-saas-server/node-app.ts -d dist --out-file crux-saas-server.js --no-minify --no-cache --detailed-report --target node --bundle-node-modules && node dist/crux-saas-server.js",
```
Comment (quixote911) [inline package.json]: Remove parcel involvement, use ts-node

**Review by quixote911 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SanchayPahalwani) [reply package.json]: Updated

**Comment by quixote911** [comment]
Lots of new code has been merged into feature/p2pmessaging, now cruxWalletClient.sendPaymentRequest and receivePaymentRequest is working perfectly.

Lets test `cruxWalletClient.sendPaymentRequest` with this server and add it to the whitelist.

**Comment by quixote911** [comment]
Looks good!

**Review by quixote911 — APPROVED** [review]

**Pull request merged**